wire-gradle-plugin/src/test/projects/java-project-java-protos/build.gradle

```diff
@@ -6,10 +6,10 @@ plugins {
 mainClassName = 'com.squareup.dinosaurs.Sample'
 
 repositories {
+  mavenLocal()
   mavenCentral()
 }
 
 dependencies {
-  implementation 'com.squareup.wire:wire-runtime:+'
-  implementation 'com.squareup.okio:okio:+'
+  implementation 'com.squareup.wire:wire-runtime:3.0.0-SNAPSHOT'
```

Comment:
Why not retain `+`? Otherwise you'll want to inject the version as a system property from the enclosing build. We do this in SQL Delight.

Reply:
See my comment above

Reply:
I would inject the version into the tests as a system property and then inject it into the Gradle project by writing it to `gradle.properties` or something.

Reply:
Yeah, it might already be injectable since it's in `gradle.properties`. Guess the problem should also disappear once we publish 3.0, `+` should start resolving to `3.1.0-SNAPSHOT` properly.

Reply:
It's in this projects `gradle.properties` but not the test project's

Reply:
PTAL 6996779